src/main/AuthenticationRequest.ts

```diff
@@ -276,7 +276,7 @@ function createClaimsPayload(opts: ClaimOpts): ClaimPayload {
         } else {
           vp_token = {
             //TODO: nonce should be initialized correctly
-            nonce: 'NONCE_STRING',
+            nonce: nonce,
```

Comment:
could be just nonce. Als remove the TODO

Reply:
todo removed